src/js/utils/browser.js

```diff
@@ -5,196 +5,198 @@
 import * as Dom from './dom';
 import window from 'global/window';
 
-const USER_AGENT = window.navigator && window.navigator.userAgent || '';
-const webkitVersionMap = (/AppleWebKit\/([\d.]+)/i).exec(USER_AGENT);
-const appleWebkitVersion = webkitVersionMap ? parseFloat(webkitVersionMap.pop()) : null;
-
 /**
  * Whether or not this device is an iPod.
  *
  * @static
- * @const
  * @type {Boolean}
  */
-export const IS_IPOD = (/iPod/i).test(USER_AGENT);
+export let IS_IPOD = false;
```

Comment:
If we'd rather these remain as `@const` in jsdoc, these could be temp vars and exported as consts later instead.

Reply:
I think that's reasonable. We can probably also cheat and declare it a const in the jsdoc.